src/lib/css/theme/_colors.scss

```diff
@@ -0,0 +1,87 @@
+.body--light {
+  --primary: #2d54cc;
+  --on-primary: #ffffff;
+  --primary-container: #dce1ff;
+  --on-primary-container: #001551;
+  --secondary: #595d72;
+  --on-secondary: #ffffff;
+  --secondary-container: #dee1f9;
+  --on-secondary-container: #161b2c;
+  --tertiary: #75546f;
+  --on-tertiary: #ffffff;
+  --tertiary-container: #ffd7f5;
+  --on-tertiary-container: #2c1229;
+  --neutral: #5e5e62;
+  --on-neutral: #ffffff;
+  --neutral-container: #e4e1e6;
+  --on-neutral-container: #1b1b1f;
+  --neutral-variant: #5d5e67;
+  --on-neutral-variant: #ffffff;
+  --neutral-variant-container: #e2e1ec;
+  --on-neutral-variant-container: #1a1b23;
+  --error: #ba1a1a;
+  --on-error: #ffffff;
+  --error-container: #ffdad6;
+  --on-error-container: #410002;
+  --surface: #fbf8fd;
+  --surface-dim: #dbd9de;
+  --surface-bright: #fbf8fd;
+  --on-surface: #1b1b1f;
+  --on-surface-variant: #45464f;
+  --surface-container-lowest: #ffffff;
+  --surface-container-low: #f5f3f7;
+  --surface-container: #efedf1;
+  --surface-container-high: #eae7ec;
+  --surface-container-highest: #e4e1e6;
+  --inverse-surface: #303034;
+  --inverse-on-surface: #f2f0f4;
+  --inverse-primary: #b7c4ff;
+  --outline: #767680;
+  --outline-variant: #c6c5d0;
+  --scrim: #000000;
+  --shadow: #000000;
+}
+
+.body--dark {
+  --primary: #b7c4ff;
+  --on-primary: #002681;
+  --primary-container: #0039b4;
+  --on-primary-container: #dce1ff;
+  --secondary: #c2c5dd;
+  --on-secondary: #2b3042;
+  --secondary-container: #424659;
+  --on-secondary-container: #dee1f9;
+  --tertiary: #e3bada;
+  --on-tertiary: #43273f;
+  --tertiary-container: #5c3d57;
+  --on-tertiary-container: #ffd7f5;
+  --neutral: #c8c6ca;
+  --on-neutral: #303034;
+  --neutral-container: #47464a;
+  --on-neutral-container: #e4e1e6;
+  --neutral-variant: #c6c5d0;
+  --on-neutral-variant: #2f3038;
+  --neutral-variant-container: #45464f;
+  --on-neutral-variant-container: #e2e1ec;
+  --error: #ffb4ab;
+  --on-error: #690005;
+  --error-container: #93000a;
+  --on-error-container: #ffdad6;
+  --surface: #131316;
+  --surface-dim: #131316;
+  --surface-bright: #39393c;
+  --on-surface: #e4e1e6;
+  --on-surface-variant: #e2e1ec;
+  --surface-container-lowest: #0e0e11;
+  --surface-container-low: #1b1b1f;
+  --surface-container: #1f1f23;
+  --surface-container-high: #2a2a2d;
+  --surface-container-highest: #39393c;
+  --inverse-surface: #e4e1e6;
+  --inverse-on-surface: #303034;
+  --inverse-primary: #2d54cc;
+  --outline: #90909a;
+  --outline-variant: #45464f;
+  --scrim: #000000;
+  --shadow: #000000;
+}
```

Comment:
Add a trailing newline please.

Reply:
Ok, it's a generated file (I should probably add a comment to stipulate it)